src/work-allocation/containers/task-assignment/task-assignment-container.component.spec.ts

```diff
@@ -113,7 +113,8 @@ describe('TaskAssignmentContainerComponent', () => {
     expect(component.caseworker).toBeUndefined();
     expect(component.error).toBeNull();
 
-    component.assign();
+
+    component.reassign();
```

Comment:
This should be `component.assign();`. I changed it to be more generic as it's for both assignment and reassignment. Please change it back to what it was.

Reply:
Ok, merge issue again